packages/contracts-bedrock/test/safe/LivenessModule2.t.sol

```diff
@@ -92,6 +54,58 @@ abstract contract LivenessModule2_TestInit is Test, SafeTestTools {
             _safe, address(livenessModule2), 0, abi.encodeCall(LivenessModule2.respond, ()), Enum.Operation.Call
         );
     }
+
+    /// @notice Helper to get the guard address from a Safe
+    function _getGuard(SafeInstance memory _safe) internal view returns (address) {
+        // keccak256("guard_manager.guard.address") from GuardManager
+        bytes32 guardSlot = 0x4a204f620c8c5ccdca3fd54d003badd85ba500436a431f0cbda4f558c93c34c8;
```

Comment:
Should we put this in `Constants.sol`? We currently have it hardcoded in 4 files